src/DEV-2/Counter.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DEV_2
+{
+    class Counter
+    {
+        public float count(List<float> numbers, ArrayList signes)
+        {
+            try
+            {
+                while (signes.IndexOf('*') != -1)
+                {
+                    int index = signes.IndexOf('*');
+                    numbers[index] = numbers[index] * numbers[index + 1];
+                    numbers.RemoveAt(index + 1);
```

Comment:
Do not repeat yourself, too much copy paste, rework (use checker class possibly)